composer.json

```diff
@@ -28,7 +28,8 @@
         "symfony/console": "^2.8 || ^3.2",
         "symfony/form": "^2.8 || ^3.2",
         "symfony/http-foundation": "^2.8 || ^3.2",
-        "symfony/security": "^2.8 || ^3.2",
+        "symfony/security-acl": "^2.8 || ^3.0",
```

Comment:
why 3.0 for this one, 3.2 for the other?

Reply:
Oh I know, it simply does not exist for acl

Reply:
@greg0ire because acl hase no 3.2 version